.github/workflows/deploy-pages.yaml

```
@@ -0,0 +1,148 @@
+name: Deploy to GitHub Pages
+
+on:
+  workflow_call:
+    inputs:
+      coverage_artifact:
+        description: 'Name of the coverage reports artifact'
+        required: true
+        type: string
+        default: 'coverage-reports'
```

actionlint (1.7.4)

input "coverage_artifact" of workflow_call event has the default value "coverage-reports", but it is also required. if an input is marked as required, its default value will never be used

(events)